src/SpectatorView.Unity/Assets/SpectatorView/Scripts/SpectatorView.cs

```diff
@@ -101,14 +103,26 @@ public class SpectatorView : MonoBehaviour
         [SerializeField]
         public float spectatorDebugVisualScale = 1.0f;
 
+        [Tooltip("Enable verbose debug logging messages")]
+        [SerializeField]
+        private bool debugLogging = false;
+
+#if UNITY_ANDROID || UNITY_IOS
         private GameObject mobileRecordingServiceVisual = null;
         private IRecordingService recordingService = null;
         private IRecordingServiceVisual recordingServiceVisual = null;
+#endif
 
         private void Awake()
         {
             Debug.Log($"SpectatorView is running as: {Role.ToString()}. Expected User IPAddress: {userIpAddress}");
 
+            GameObject settings = Resources.Load<GameObject>(SettingsPrefabName);
+            if (settings != null)
+            {
+                Instantiate(settings, null);

```

Comment:
This instantiates an object and then looses track of it, we should cache it and clean it up in OnDestroy i think